pkg/brownfield/brownfield_test.go

```diff
@@ -0,0 +1,18 @@
+// -------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// --------------------------------------------------------------------------------------------
+
+package brownfield
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestAppgw(t *testing.T) {
```

Comment:
run ginkgo bootstrap

Reply:
Didn't know that existed! Great tip!
Turns out the output of it is not too different from what was here already.
I merged the changes into this file (and renamed it).